src/util/file/file_test.go

```diff
@@ -62,12 +62,14 @@ func TestBuildDataDirDotOk(t *testing.T) {
 	require.True(t, strings.HasSuffix(builtDir, cleanDir))
 
 	gopath := os.Getenv("GOPATH")
+	// by default go uses GOPATH=$HOME/go if it is not set
 	if gopath == "" {
-		require.Equal(t, cleanDir, builtDir)
-	} else {
-		require.True(t, strings.HasPrefix(builtDir, gopath))
-		require.NotEqual(t, builtDir, filepath.Clean(gopath))
+		home := filepath.Clean(UserHome())
+		gopath = home + "/" + "go"
```

Comment:
It's better to use `filepath.Join(home, "go") to be able to compatible with different systems.

Reply:
Oh right! Thnx!